ReSwift/CoreTypes/Subscription.swift

```diff
@@ -80,6 +80,13 @@ public class Subscription<State> {
         return self._select(selector)
     }
 
+    public func select<Substate: Equatable>(
+        _ selector: @escaping (State) -> Substate
+        ) -> Subscription<Substate>
+    {
+        return self._select(selector).skipRepeats()
```

Comment:
I'm not sure if this is handled in another place, but shouldn't we check if `subscriptionsAutomaticallySkipRepeats` is true before skipping?